src/main/java/com/gregtechceu/gtceu/api/pattern/MultiblockState.java

```diff
@@ -145,22 +229,42 @@ public BlockState getOffsetState(Direction face) {
         return world.getBlockState(this.pos.relative(face));
     }
 
-    public Level getWorld() {
-        return world;
-    }
-
+    /**
+     * Adds given {@link BlockPos} to the cache
+     * 
+     * @param pos the position to add
+     */
     public void addPosCache(BlockPos pos) {
         cache.add(pos.asLong());
     }
 
+    /**
+     * Checks for the given {@link BlockPos} in the cache
+     * 
+     * @param pos the BlockPos to check
+     * @return whether the position is found
+     */
```

Comment:
The name is clear enough as to what the method does - no need for the summary.
```suggestion
    /**
     * @param pos the BlockPos to check
     * {@return {@code true} if the cache contains the given pos}
     */
```